SAMtemplates/load_balancer/main.yaml

```diff
@@ -137,6 +155,184 @@ Resources:
+        - FARGATE
+      ExecutionRoleArn: !GetAtt ECSTaskExecutionRole.Arn
+      ContainerDefinitions:
+        - Name: !Sub "${StackName}-container"
+          Image: !Sub "${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/fhir-facade-repo:${VersionNumber}"
+          PortMappings:
+            - ContainerPort: !Ref ContainerPort
+              Protocol: tcp
+          Environment:
+            - Name: VALIDATOR_HOST
+              Value: !Sub "${StackName}-validator"
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref LogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: ecs
+        - Name: !Sub "${StackName}-validator"
+          Image: !Sub "${AWS::AccountId}.dkr.ecr.${AWS::Region}.amazonaws.com/validator-repo:${VersionNumber}"
+          PortMappings:
+            - ContainerPort: !Ref ContainerPortValidator
+              Protocol: tcp
+          LogConfiguration:
+            LogDriver: awslogs
+            Options:
+              awslogs-group: !Ref LogGroup
+              awslogs-region: !Ref AWS::Region
+              awslogs-stream-prefix: ecs
+      RuntimePlatform:
+        CpuArchitecture: X86_64
+        OperatingSystemFamily: LINUX
+
+  # ECS Service Configuration
+  ECSService:
+    Type: AWS::ECS::Service
+    DependsOn: ListenerHTTPS
+    Properties:
+      Cluster: !Ref ECSCluster
+      ServiceName: !Sub "${StackName}-service"
+      DesiredCount:
+        !If
+          - IsRefOrProd
+          - 2
+          - 1
+      LaunchType: FARGATE
+      NetworkConfiguration:
+        AwsvpcConfiguration:
+          AssignPublicIp: DISABLED
+          SecurityGroups:
+            - !Ref ContainerSecurityGroup
+          Subnets: !Split [",", !ImportValue vpc-resources:PrivateSubnets]
+      TaskDefinition: !Ref ECSTaskDefinition
+      LoadBalancers:
+        - ContainerName: !Sub "${StackName}-container"
+          ContainerPort: !Ref ContainerPort
+          TargetGroupArn: !Ref TargetGroup
+
+  # ECS Service Auto Scaling Role
+  ECSServiceAutoScalingRole:
+    Type: AWS::IAM::Role
+    Condition: IsRefOrProd
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service: application-autoscaling.amazonaws.com
+            Action: sts:AssumeRole
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/service-role/AmazonEC2ContainerServiceAutoscaleRole
+      RoleName: !Sub "${StackName}-ecsAutoScalingRole"
+
+  # ECS Service Auto Scaling Target
+  ECSServiceAutoScalingTarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    Condition: IsRefOrProd
+    DependsOn: ECSService
+    Properties:
+      MinCapacity:
+        !If
+          - IsRefOrProd
+          - 2
+          - 1
+      MaxCapacity:
+        !If
+          - IsRefOrProd
+          - 10
+          - 6
```

Comment:
no need for these conditions as the resource isn't even there for non ref and prod